Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:_GOVERSION_
```

Comment:
Can you move this under a `build` directory please. Since its for a build/test image.